src/UNIV2LPOracle.t.sol

```diff
@@ -363,6 +363,68 @@ contract UNIV2LPOracleTest is DSTest {
         assertEq(rootVal, rootAltVal);
     }
 
+
+
+    uint112 max112 = type(uint112).max;
+
+    function test_seek_equivalence(
+        uint8 decimals0,
+        uint8 decimals1,
+        uint112 res0,
+        uint112 res1
+    ) public {
+        uint256 dec0 = uint256(decimals0);
+        uint256 dec1 = uint256(decimals1);
+        if (dec1 > 18 || dec1 < 1) {
+            return;
+        }
+        if (dec0 > 18 || dec0 < 1) {
+            return;
+        }
```

Comment:
This is kind of inefficient, since assuming independent and uniformly random distributions for `decimals0` and `decimals1`, the test only passes these checks (17/256)^2 ~ 0.44% of the time. An improvement would be to set `dec0` to `decimals0 % 32` (and similarly for `dec1`). Since `32 | 256`, you aren't introducing any bias into the distribution, and the test now passes the checks (17/32)^2 ~ 28% of the time, a 63x improvement.

Reply:
To clarify:
Are you saying that the current test by purely random fuzzing "wastes" many of its runs, and by transforming the inputs % 32, more possible inputs get mapped into the acceptable range?

If so, great observation & I agree. 

If not, then I will need further explanation.

Reply:
Yep, that's what I meant.